input/drivers_joypad/xinput_hybrid_joypad.c

```diff
@@ -611,13 +611,28 @@ static void xinput_joypad_poll(void)
          xinput_hotplug_index = (xinput_hotplug_index + 1) % 4;
    }
 
+   bool has_active_ports = false;
```

Comment:
The project expects C89 compatibility, declarations need to be at the top of the block.